### DISCUSSION TITLE
Folder delete - cascade content

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9731] [Feature Requests]
As thinking of "folders" from desktops, i though that all the contents childs are also removed. Seems like this is not the case. Is there a setting for this behavior ?

**Comment by benhaynes** [comment]
It's not yet, but we should make this an option. Also, this ties into the nested folder permissions discussion too.

**Reply by NilsBaumgartner1994** [reply]
Would be very nice, think about selective cascade or atleast an optional parameter.

Orienting at how desktop ui are doing it

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
